examples/experimental/onnx/mobilenet_v2/main.py

```diff
@@ -0,0 +1,168 @@
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import time
+from pathlib import Path
+from typing import Dict, List
+
+import onnx
+import onnxruntime as rt
+import torch
+import torchvision
+
+import nncf
+from examples.experimental.onnx.mobilenet_v2 import utils
+
+# Path to the `mobilenet_v2` directory.
+ROOT = Path(__file__).parent.resolve()
+# Path to the directory where the original and quantized models will be saved.
+MODEL_DIR = ROOT / 'mobilenet_v2_quantization'
+# Path to ImageNet validation dataset.
+DATASET_DIR = Path("/ssd/imagenet") #ROOT / 'imagenet'
+
+
+def run_example():
+    """
+    Runs the MobileNetV2 quantization example.
+    """
+    # Step 1: Instantiate the MobileNetV2 from the PyTorch Hub.
+    # For more details, please see the [link](https://pytorch.org/hub/pytorch_vision_mobilenet_v2).
+    model = torch.hub.load('pytorch/vision:v0.10.0', 'mobilenet_v2', pretrained=True)
+    model.eval()
+
+    # Step 2: Export the PyTorch model to ONNX.
+    if not MODEL_DIR.exists():
+        os.makedirs(MODEL_DIR)
+
+    onnx_model_path = MODEL_DIR.joinpath('mobilenet_v2.onnx')
+    dummy_input = torch.randn(1, 3, 224, 224)
+    torch.onnx.export(model, dummy_input, onnx_model_path, verbose=False)
+
+    # Step 3: Load the ONNX model.
+    onnx_model = onnx.load(onnx_model_path)
+
+    # Step 4: Create calibration dataset.
+    data_source = create_data_source()
+
+    # Step 5: Apply quantization algorithm.
+
+    # Define the transformation method. This method should take a data item returned
+    # per iteration through the `data_source` object and transform it into the model's
+    # expected input that can be used for the model inference.
+    input_name = onnx_model.graph.input[0].name
+    def transform_fn(data_item):
+        images, _ = data_item
+        return {input_name: images.numpy()}
+
+    # Wrap framework-specific data source into the `nncf.Dataset` object.
+    calibration_dataset = nncf.Dataset(data_source, transform_fn)
+
+    # Quantization of the ONNX model. The `quantize` method expects an ONNX
+    # model as input and returns an ONNX model that has been quantized using
+    # the calibration dataset.
+    quantized_model = nncf.quantize(onnx_model, calibration_dataset)
+
+    # Step 6: Save the quantized model.
+    quantized_model_path = MODEL_DIR / 'quantized_mobilenet_v2.onnx'
+    onnx.save(quantized_model, quantized_model_path)
+    print(f"The quantized model has been saved in: {quantized_model_path}")
```

Comment:
```suggestion
    print(f'The quantized model has been saved in: {quantized_model_path}')
```
Could you please do this for consistency with the rest of the file?